websites/T/TRanimeizle/metadata.json

```
@@ -11,10 +11,10 @@
 		"nl": "TRanimeizle is een Turkse anime-website om online animaties te zoeken en te bekijken."
 	},
 	"url": [
-		"www.tranimeizle.net",
-		"tranimeizle.net"
+		"www.tranimeizle.co",
+		"tranimeizle.co"
 	],
-	"version": "1.0.11",
+	"version": "1.0.12",
 	"logo": "https://i.imgur.com/TsLirN4.jpg",
```

Higher quality updated logo. (Also change in presence.ts please.)
```suggestion
	"logo": "https://i.imgur.com/BnMRZ49.png",
```

---

I'll do asap